app/views/layouts/application.html.erb

```diff
@@ -4,9 +4,9 @@
     <title>Airbnb</title>
```

Comment:
- Fix indentation for this entire file.
- Indent contents of the `body` element:
![image](https://user-images.githubusercontent.com/20994464/63744801-0770d300-c8d3-11e9-8382-40d5f291970d.png)

Additional note: You can move the scripts to the js files in the assets folder.